kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -50,7 +50,7 @@ define([
 
         init: function (options) {
             this._super(options);
-            this.bulkImportTypes = ['fastq_reads'];
+            this.bulkImportTypes = ['fastq_reads', 'sra_reads'];
```

Comment:
Now that we're detaching some of the dummy data, we need to allow `sra_reads` as well. We can also add others as we go.